pocketmine-mp/Dockerfile

```diff
@@ -59,4 +59,4 @@ EXPOSE 19132/udp
 
 VOLUME ["/data", "/plugins"]
 
-CMD start-pocketmine
+CMD ["start-pocketmine"]
```

Comment:
why? this isn't related to the bug fix, does it?

Reply:
Without square bracket, It will be started as a subcommand of `/bin/sh -c`, which does not pass signals. It must be *exec form*(means square brackets) to pass signals like `SIGTERM`.

Reply:
@SOF3 pls review this, I don't know enough about this to comment

Reply:
I think this part is fine.